### ISSUE TITLE
Event and State Records not created

### ISSUE THREAD

**Issue by tobyaharris** [DigitalShoestringSolutions/LocationTracking#19]
I'm trying to test this solution to take some screenshots for the README for PR 9. 

TLDR; I can't get it to work. 

I have created a few locations, products and product types in :8002. 
I have created a barcode identifier for each individual product and each product type. 
From :80 I scan products / qtys of prodtypes between locs and this input seems to be accepted
![image](https://github.com/user-attachments/assets/5460744b-9f6d-4141-b7e5-d2182af181ed)

At :8000 I see a dashboard with colums for the locations I created, but no entries in the table.
![image](https://github.com/user-attachments/assets/64ba6cea-e8b8-4a03-96ad-fee826c9579b)

I've gone to :8000/settings and reset any view customisation
The Item Name Cache has size: 0 so can't be cleared. 
I've closed and reopend the browser
I've stopped, rebuilt and restarted the solution. 

I have logged into to :8001 and can see there are 0 Event Records and 0 State Records. Are not some meant to be created when items are moved between locations?

What am I missing?
(Besides the source code documentation, that I'm trying to create 😃 )

**Comment by tobyaharris** [comment]
Get some log files. Possibly MQTT failure?

**Comment by tobyaharris** [comment]
I don't have logs to hand, they have likely already been deleted. Nor do I have motivation to dig for them in case they are recoverable. Closing as can't reproduce, but keeping the record in case anything similar happens in future.
